src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt

```diff
@@ -431,4 +431,49 @@ MembersMustExist : Member 'System.Threading.Tasks.Task.Dispose(System.Boolean)'
 CannotRemoveBaseTypeOrInterface : Type 'System.Threading.Tasks.Task<TResult>' does not implement interface 'System.IDisposable' in the implementation but it does in the contract.
 MembersMustExist : Member 'System.Attribute.IsDefaultAttribute()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Attribute.Match(System.Object)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Attribute.TypeId.get()' does not exist in the implementation but it does exist in the contract.
\ No newline at end of file
+MembersMustExist : Member 'System.Attribute.TypeId.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
By adding SystemException in System.Runtime uap10.1 library, do you mean adding a stub for it? I'm asking because adding that still doesn't get rid of ApiCompat errors like:
```
CannotRemoveBaseTypeOrInterface : Type 'System.ArgumentException' does not inherit from base type 'System.SystemException' in the implementation but it
does in the contract.
```